src/Zynga/Framework/PgData/V1/PgModel/Writer.hh

```diff
@@ -43,15 +43,32 @@ class Writer implements WriterInterface {
     list($isDefaultValue, $isDefaultError) = $pk->isDefaultValue();
 
     if ($isDefaultValue === true) {
+      $retryOnFailure = true;
```

Comment:
While it is unlikely this could cause an infinite retry loop, do we want to have some cap to the number of retries?